theseus/optimizer/nonlinear/levenberg_marquardt.py

```diff
@@ -9,10 +9,20 @@
 
 from theseus.core import Objective
 from theseus.optimizer import Linearization
-from theseus.optimizer.linear import DenseSolver, LinearSolver
+from theseus.optimizer.linear import DenseSolver, LinearSolver, LUCudaSparseSolver
 
 from .nonlinear_least_squares import NonlinearLeastSquares
 
+_LM_ALLOWED_SOLVERS = [DenseSolver, LUCudaSparseSolver]
```

Comment:
Might be confusing name if you intend to use this for ellipsoidal damping and the list is different for default/spherical damping.

Reply:
Good point!

Reply:
Fixed.